src/utils/extractParentsFromRule.tsx

```diff
@@ -0,0 +1,18 @@
+import _ from "lodash";
+
```

Comment:
Probable rename as ``extractChildrenFromRule`` and the file could be something like ``utils/metadataIncludeExclude.ts`` (note that this is not a react file so it shouldn't be TSX.

```
const childrenFromRule = (rule: string) =>
                oldIncludeRules.filter((children: string) => children.includes(rule + "."));
```

Reply:
I did not move childrenFromRule to utils because oldIncludeRules is a variable of moveIncludeExcludeRules method in SyncRule, then seems not very util this function in a separate file under utils folder.

Reply:
Understood, yeah I meant increasing the parameters ``(allRules: string[], rule: string)``. But it's your call! :)

Reply:
Done!